src/geo/map/cartodb-layer.js

```diff
@@ -31,6 +45,17 @@ var CartoDBLayer = LayerModelBase.extend({
     return this.get('visible');
   },
 
+  getGeometryType: function () {
```

Comment:
AFIK this is used [here](https://github.com/CartoDB/deep-insights.js/pull/304/files#diff-a7270c805528a9245fa1351d6719bcb2R7) to determine the cartocss that should be generated, based on the type of geometry... If getting the type of geometry is not doable, perhaps we can generate a more generic cartocss such as:

```
// points
#layer_name['mapnik::geometry_type'=1] {
  ...
}
// lines
#layer_name['mapnik::geometry_type'=2] {
  ...
}
// polygons
#layer_name['mapnik::geometry_type'=3] {
  ..
}
```